tests/cli/set/test_samples.py

```diff
@@ -0,0 +1,138 @@
+"""Test methods for cg/cli/set/samples"""
+import pytest
+
+from cg.cli.set import samples
+from cg.store import Store
+
+SUCCESS = 0
+
+
+@pytest.mark.parametrize("identifier_key", ["name", "internal_id"])
+def test_set_samples_by_identifiers(
+    cli_runner, base_context, base_store: Store, identifier_key, helpers
+):
+    # GIVEN a database with a sample
```

Comment:
Should not the tests check if the original state was altered?

Reply:
no, this method changes nothing :)

Reply:
it invokes set sample that alters things, and should be tested for that

Reply:
Ok, so I guess `set_samples` is tested somewhere else then?

Reply:
yes in the file test_set_sample.py :)

Reply:
test_sample.py it is

Reply:
Perfect!